exercises/clock/clock_test.rb

```diff
@@ -5,281 +5,313 @@
 
   def test_hour_rolls_over
     skip
-    assert_equal "01:00", Clock.at(25, 0).to_s
+    assert_equal "01:00", Clock.new(hour: 25, minute: 0).to_s
   end
 
   def test_hour_rolls_over_continuously
     skip
-    assert_equal "04:00", Clock.at(100, 0).to_s
+    assert_equal "04:00", Clock.new(hour: 100, minute: 0).to_s
   end
 
   def test_sixty_minutes_is_next_hour
     skip
-    assert_equal "02:00", Clock.at(1, 60).to_s
+    assert_equal "02:00", Clock.new(hour: 1, minute: 60).to_s
   end
 
   def test_minutes_roll_over
     skip
-    assert_equal "02:40", Clock.at(0, 160).to_s
+    assert_equal "02:40", Clock.new(hour: 0, minute: 160).to_s
   end
 
   def test_minutes_roll_over_continuously
     skip
-    assert_equal "04:43", Clock.at(0, 1723).to_s
+    assert_equal "04:43", Clock.new(hour: 0, minute: 1723).to_s
   end
 
   def test_hour_and_minutes_roll_over
     skip
-    assert_equal "03:40", Clock.at(25, 160).to_s
+    assert_equal "03:40", Clock.new(hour: 25, minute: 160).to_s
   end
 
   def test_hour_and_minutes_roll_over_continuously
     skip
-    assert_equal "11:01", Clock.at(201, 3001).to_s
+    assert_equal "11:01", Clock.new(hour: 201, minute: 3001).to_s
   end
 
   def test_hour_and_minutes_roll_over_to_exactly_midnight
     skip
-    assert_equal "00:00", Clock.at(72, 8640).to_s
+    assert_equal "00:00", Clock.new(hour: 72, minute: 8640).to_s
   end
 
   def test_negative_hour
     skip
-    assert_equal "23:15", Clock.at(-1, 15).to_s
+    assert_equal "23:15", Clock.new(hour: -1, minute: 15).to_s
   end
 
   def test_negative_hour_rolls_over
     skip
-    assert_equal "23:00", Clock.at(-25, 0).to_s
+    assert_equal "23:00", Clock.new(hour: -25, minute: 0).to_s
   end
 
   def test_negative_hour_rolls_over_continuously
     skip
-    assert_equal "05:00", Clock.at(-91, 0).to_s
+    assert_equal "05:00", Clock.new(hour: -91, minute: 0).to_s
   end
 
   def test_negative_minutes
     skip
-    assert_equal "00:20", Clock.at(1, -40).to_s
+    assert_equal "00:20", Clock.new(hour: 1, minute: -40).to_s
   end
 
   def test_negative_minutes_roll_over
     skip
-    assert_equal "22:20", Clock.at(1, -160).to_s
+    assert_equal "22:20", Clock.new(hour: 1, minute: -160).to_s
   end
 
   def test_negative_minutes_roll_over_continuously
     skip
-    assert_equal "16:40", Clock.at(1, -4820).to_s
+    assert_equal "16:40", Clock.new(hour: 1, minute: -4820).to_s
   end
 
   def test_negative_hour_and_minutes_both_roll_over
     skip
-    assert_equal "20:20", Clock.at(-25, -160).to_s
+    assert_equal "20:20", Clock.new(hour: -25, minute: -160).to_s
   end
 
   def test_negative_hour_and_minutes_both_roll_over_continuously
     skip
-    assert_equal "22:10", Clock.at(-121, -5810).to_s
+    assert_equal "22:10", Clock.new(hour: -121, minute: -5810).to_s
   end
 
   def test_add_minutes
     skip
-    assert_equal "10:03", (Clock.at(10, 0) + 3).to_s
+    clock = Clock.new(hour: 10, minute: 0)
+    clock + Clock.new(minute: 3)
```

Comment:
`+` should still work if a clock is immutable. So I'd put the wrapping parentheses back.

 assert_equal "10:03", (clock + Clock.new(minute: 3)).to_s